courses/models_test.py

```diff
@@ -1,12 +1,17 @@
 """Tests for course models"""
 from datetime import timedelta
 
+import json
 import pytest
 import factory
 
 from courses.factories import ProgramFactory, CourseFactory, CourseRunFactory
 from ecommerce.factories import ProductFactory, ProductVersionFactory
-from cms.factories import ProgramPageFactory, CoursePageFactory
+from cms.factories import (
+    ProgramPageFactory,
+    CoursePageFactory,
+    WhoShouldEnrollPageFactory,
+)
```

Comment:
I am moving the test* files under `/courses/tests/` in my [PR](https://github.com/mitodl/mitxpro/pull/170)

Reply:
Okay, I'll update my files once #170 goes through.